### PR TITLE
Try some ramda

### PR DESCRIPTION
**Pull request by achiu8**
Trying out some ramda.. what do you think?

### CODE REVIEW

File: models/order.js [hunk 1]
```diff
@@ -25,18 +25,11 @@ orderSchema.statics.getOrders = function() {
 orderSchema.statics.updateEntry = function(orderId, entryIndex, update) {
   return this.findOne({ id: orderId })
     .then((order) => {
```
Comment (achiu8) [inline models/order.js:27]: I think this is pretty cool, but maybe a little convoluted..

File: src/components/Chef.js [hunk 2]
```diff
@@ -35,10 +35,12 @@ class Chef extends Component {
   }
 
   getOpenEntries() {
```
Comment (achiu8) [inline src/components/Chef.js:37]: This is probably the coolest.

File: src/components/Chef.js [hunk 3]
```diff
@@ -35,10 +35,12 @@ class Chef extends Component {
...
-    const allEntries = this.props.orders.reduce((entries, order) => entries.concat(toOpenEntries(order)), []);
-    const sortedOpenEntries = _.sortBy(allEntries.filter(({ entry }) => entry.status === constants.OPEN), 'createdAt');
-
-    return sortedOpenEntries.slice(0, displayMax);
+    return _.pipe(
+      _.chain(toOpenEntries),
+      _.filter(_.pathEq(['entry', 'status'], constants.OPEN)),
+      _.sortBy(_.prop('createdAt')),
+      _.take(displayMax)
+    )(this.props.orders);
```
Comment (kprakobkit) [inline src/components/Chef.js:43]: Nice. Really readable.

File: models/order.js (hunk 1 above)
Comment (kprakobkit) [inline models/order.js:27]: apparently ramdajs site is not responsive. I'll look at this more later...

File: src/components/Entry.js [hunk 4]
```diff
@@ -16,7 +16,7 @@ class Entry extends Component {
   }
 
   showDelivered() {
-    return this.props.status === constants.OPEN || this.props.status === constants.COMPLETED;
+    return _.contains(this.props.status, [constants.OPEN, constants.COMPLETED]);
```
Comment (kprakobkit) [inline src/components/Entry.js:19]: Oh thank god

File: models/order.js (hunk 1 above)
Comment (kprakobkit) [inline models/order.js:27]: Yo good explanation of underscore vs ramda, also pCompose?? http://developer.telerik.com/featured/practical-functional-javascript-ramda/

Comment (kprakobkit) [inline models/order.js:27]: From the documentation, there's composeP and pipeP...nice

Comment (achiu8) [inline models/order.js:27]: Yeah, I tried using those in some of the places where we chain a lot of `then`s, but I think the ramda functions require every argument to return a promise, so we'd have to explicitly wrap the ones where we are just doing some transforms (like `toEntry` or something).

**Comment by kprakobkit** [comment]
Good to merge?

**Comment by achiu8** [comment]
Yeah, if you are cool with using it.

**Pull request merged**